### PR TITLE
Overhaul Stirr & Add Wise TV

### PR DESCRIPTION
**Pull request by brokenstream**
Nearly every link leads to 404.

### CODE REVIEW

File: streams/us_stirr.m3u [hunk 1]
```diff
@@ -1,4 +1,16 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="24HourFreeMovies.us",24 Hour Free Movies (720p)
+https://d1j2u714xk898n.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-24HourFreeMovies/145.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:3]: Basically is just https://d1j2u714xk898n.cloudfront.net/scheduler/scheduleMaster/145.m3u8

File: streams/us_stirr.m3u [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+#EXTINF:-1 tvg-id="",4K Travel TV (1080p)
+https://streams2.sofast.tv/ptnr-stirr/genre-Travel/lang-English/ctgr-Travel/title-4K-TRAVEL-TV-ENG/sofastplayout/33c31ac4-51fa-46ae-afd0-0d1fe5e60a80_0_HLS/master.m3u8
+#EXTINF:-1 tvg-id="AdventureSportsTV.us",Adventure Sports TV (1080p)
+https://d3rl6ns7c3ilda.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-AdventureSportsTV/438.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:7]: https://d3rl6ns7c3ilda.cloudfront.net/scheduler/scheduleMaster/438.m3u8

File: streams/us_stirr.m3u [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+#EXTINF:-1 tvg-id="",Always Funny: Pranks & Fails (720p)
+https://linear-863.frequency.stream/dist/stirr/863/hls/master/playlist.m3u8
+#EXTINF:-1 tvg-id="AlwaysFunnyVideos.us",Always Funny Videos (720p)
+https://linear-12.frequency.stream/dist/stirr/12/hls/master/playlist.m3u8
+#EXTINF:-1 tvg-id="",Biz Talk Today TV (1080p)
+https://d3htp73xsa9p15.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-BizTalk/445.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:13]: https://d3htp73xsa9p15.cloudfront.net/scheduler/scheduleMaster/445.m3u8

File: streams/us_stirr.m3u [hunk 4]
```diff
@@ -19,239 +31,123 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
 https://dai.google.com/linear/hls/event/Xu-ITJ2GTNGaxGn893mmWg/master.m3u8
 #EXTINF:-1 tvg-id="CBSNewsPittsburgh.us",CBSN Pittsburgh PA (720p)
 https://dai.google.com/linear/hls/event/i5SXVKI4QIuV-eF2XAH4FQ/master.m3u8
+#EXTINF:-1 tvg-id="",Channel Fight (1080p)
+https://d15wqvt0xm15k4.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/ChannelFight-Nuestra/266.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:35]: https://d15wqvt0xm15k4.cloudfront.net/scheduler/scheduleMaster/266.m3u8

That's a whole bunch of MuxIP streamers you've stumbled upon.

File: streams/us_stirr.m3u [hunk 5]
```diff
@@ -19,239 +31,123 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
...
-#EXTINF:-1 tvg-id="DoveChannel.us",Dove Channel (1080p)
-https://dai.google.com/linear/hls/event/dfbBGQhPQQqypdEAjpUGlA/master.m3u8
+#EXTINF:-1 tvg-id="",Cowboy Movie Channel (1080p)
+https://streams2.sofast.tv/ptnr-stirr/genre-Entertainment/lang-English/ctgr-Movies/title-COWBOY-MOVIE-CHANNEL-ENG/sofastplayout/32eb332e-f644-46e5-ad91-e55ad80d14f7_0_HLS/master.m3u8
+#EXTINF:-1 tvg-id="",Craftsy (1080p)
+https://linear-492.frequency.stream/dist/stirr/492/hls/master/playlist.m3u8
+#EXTINF:-1 tvg-id="",Crafty Panda (720p)
+https://bpd-cp-stirr.otteravision.com/bpd/cp/cp.m3u8
+#EXTINF:-1 tvg-id="",Cricket Gold (1080p)
+https://d382r3rgbxdixq.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-CricketGold/418.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:47]:
```suggestion
https://d382r3rgbxdixq.cloudfront.net/scheduler/scheduleMaster/418.m3u8
```

File: streams/us_stirr.m3u [hunk 6]
```diff
@@ -19,239 +31,123 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
...
+#EXTINF:-1 tvg-id="",CrimeFlix (1080p)
+https://dgx2snzx175pa.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-CrimeFlix/391.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:49]: https://dgx2snzx175pa.cloudfront.net/scheduler/scheduleMaster/391.m3u8

File: streams/us_stirr.m3u [hunk 7]
```diff
@@ -19,239 +31,123 @@ http://dai.google.com/linear/hls/event/rtcMlf4RTvOEkaudeany5w/master.m3u8
...
+#EXTINF:-1 tvg-id="FITE247.us",FITE 24/7 (1080p)
+https://d3d85c7qkywguj.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/FiteTV-Nuestra/263.m3u8
+#EXTINF:-1 tvg-id="FreebieTV.us",Freebie TV (720p)
+https://d26pp4b7ojtlyn.cloudfront.net/v1/master/9d062541f2ff39b5c0f48b743c6411d25f62fc25/STIRR-MuxIP-FreebieTV/187.m3u8
+#EXTINF:-1 tvg-id="FUELTV.at",Fuel TV (1080p)
+https://amg01074-fueltv-amg01074c1-stirr-us-4214.playouts.now.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="GoTraveler.us",GoTraveler (1080p)
+https://amg09501-questar-amg09501c1-stirr-us-3094.playouts.now.amagi.tv/amg09501/AMG09501C1/segmented_playlist/0/35-89-3094.m3u8
+#EXTINF:-1 tvg-id="",Hipstr (1080p)
+https://amg09501-questar-amg09501c2-stirr-us-3095.playouts.now.amagi.tv/amg09501/AMG09501C2/segmented_playlist/0/35-89-3095.m3u8
```
Comment (BellezaEmporium) [inline streams/us_stirr.m3u:85]: Don't know how you managed to get those ones but let me just say that's brilliant.

**Review by BellezaEmporium — APPROVED** [review]
LGTM.

File: streams/us_stirr.m3u (hunk 4 above)
Comment (brokenstream) [reply streams/us_stirr.m3u:35]: Yeah, I was debating whether or not to use the "scheduleMaster" links, as many of them are for other services like Distro, but later decided to just keep them specific to Stirr for now.

File: streams/us_stirr.m3u (hunk 7 above)
Comment (brokenstream) [reply streams/us_stirr.m3u:85]: Can't take credit for that, as I got them from Matt Huisman's updated playlist.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**